tests/docker_images/linux-loongarch64/ubuntu-x-tools/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR ISC
+
+FROM ubuntu:latest
+
+RUN apt-get update && apt-get install -y  \
+    git cmake golang gdb gdbserver valgrind \
+    libclang1 \
+    build-essential \
+    ssh \
+    rsync \
+    tar \
+    python3 \
+    xz-utils \
+    ninja-build
+
+RUN apt-get install -y \
+    qemu-user \
+    qemu-user-binfmt \
+    qemu-user-status
+# Not yet available
+#    qemu-system-loongarch64
+
+RUN apt-get clean
+
+COPY loongarch64-x-tools.tar.xz /
+RUN tar Jxvf loongarch64-x-tools.tar.xz -C / && rm /loongarch64-x-tools.tar.xz
+COPY loongarch64.cmake /
+
+EXPOSE 1234
```

Comment:
what is this port used for?

Reply:
It's for connecting gdb for debugging. More specifically, the image is setup so that you could run [Qemu "user space" emulation](https://www.qemu.org/docs/master/user/main.html) with a `-g 1234` parameter to have it listen for a debugger.